cli/compile/compile.go

```diff
@@ -73,8 +73,8 @@ func NewCommand() *cobra.Command {
 	command.Flags().BoolVarP(&dryRun, "dry-run", "n", false, "Perform the build but do not copy the compile output file.")
 	command.Flags().StringVar(&buildPath, "build-path", "",
 		"Path where to save compiled files. If omitted, a directory will be created in the default temporary path of your OS.")
-	command.Flags().StringSliceVar(&buildProperties, "build-properties", []string{},
-		"List of custom build properties separated by commas. Or can be used multiple times for multiple properties.")
+	command.Flags().StringArrayVar(&buildProperties, "build-properties", []string{},
+		"List of custom build properties separated by spaces. Or can be used multiple times for multiple properties.")
```

Comment:
Since if you want to change multiple properties you now need to repeat the flag multiple times I think that the correct description is something along the line of:

```suggestion
		"Override a build property with a custom value. Can be used multiple times for multiple properties.")
```